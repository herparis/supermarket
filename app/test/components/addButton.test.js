import React from 'react'
import { shallow } from 'enzyme'

import AddButton from '../../src/components/addButton/addButton'

function checkRender(wrapper) {
  expect(wrapper.isEmptyRender()).toBe(false)
  expect(wrapper).toMatchSnapshot()
}

describe('AddButton components rendeding', () => {
  const addButton = shallow(<AddButton />)
  it('addButton renders', () => {
    checkRender(addButton)
  })
})