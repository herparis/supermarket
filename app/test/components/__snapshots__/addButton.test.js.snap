// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddButton components rendeding addButton renders 1`] = `ShallowWrapper {}`;
